github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -797,6 +794,39 @@ periodics:
       resources:
         requests:
           memory: "200Mi"
+- name: periodic-project-infra-gcp-cluster-deprovision
+  cron: "45 7 * * *"
+  max_concurrency: 1
+  annotations:
+    testgrid-create-test-group: "false"
+  labels:
+    preset-gcs-credentials: "true"
```

Comment:
I faintly remember that the service account credentials we are using might only have gcs access. Is there a way to check access rights of service credentials?

We might want to create a new service account anyway - we could also sure that it only has access to resources we really want to delete.

Reply:
I've updated it to use the new service account from - https://github.com/kubevirt/project-infra/pull/3787